### PR TITLE
Add JDBC Assist for Presto

### PR DESCRIPTION
**Pull request by kokosing**
Add JDBC Assist for Presto

### CODE REVIEW

**Comment by electrum** [comment]
Fixes #799

File: desktop/libs/notebook/src/notebook/connectors/base.py [hunk 1]
```diff
@@ -359,13 +359,19 @@ def get_api(request, snippet):
...
       from notebook.connectors.jdbc_teradata import JdbcApiTeradata
       return JdbcApiTeradata(request.user, interpreter=interpreter)
+    elif interpreter['options'] and interpreter['options'].get('url', '').find('presto') >= 0:
+      from notebook.connectors.jdbc_presto import JdbcApiPresto
+      return JdbcApiPresto(request.user, interpreter=interpreter)
+    else:
+      from notebook.connectors.jdbc import JdbcApi
+      return JdbcApi(request.user, interpreter=interpreter)
   elif interface == 'teradata':
+    from notebook.connectors.jdbc_presto import JdbcApiPresto
```
Comment (electrum) [inline desktop/libs/notebook/src/notebook/connectors/base.py]: This block appears to be reversed with the next one

**Comment by romainr** [comment]
+1 for above and would it be possible to add a commented example in the hue.ini like https://github.com/cloudera/hue/blob/master/desktop/conf.dist/hue.ini#L839 and also in https://github.com/cloudera/hue/blob/master/desktop/conf/pseudo-distributed.ini.tmpl (that way it documents it better until we have proper documentation)

**Comment by romainr** [comment]
Could also add a note to https://github.com/cloudera/hue/blob/master/docs/user-guide/user-guide.md#presto in another PR if you want

**Comment by kokosing** [comment]
Thank you for a review! Comments addressed.

**Comment by romainr** [comment]
Thanks a lot!

**Comment by kokosing** [comment]
Thanks!

**Pull request merged**